src/web/mjs/connectors/LittleGarden.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class LittleGarden extends Connector {
+    constructor() {
+        super();
+        super.id = 'littlegarden';
+        super.label = 'Little Garden';
+        this.tags = ['manga', 'french'];
+        this.url = 'https://littlexgarden.com';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, 'h2.super-title');
+        const id = uri.pathname;
+        const title = data[0].innerText.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        const request = new Request(new URL('/mangas', this.url), this.requestOptions);
+        const urlList = await this.fetchDOM(request, 'div.listing a.no-select');
+
+        return urlList.map((urlElement) => ({
+            id: this.getRelativeLink(urlElement),
```

Comment:
May use `this.getRootRelativeOrAbsoluteLink(urlElement, this.url)` instead of deprecated `getRelativeLink`